Modix/Modix.csproj

```diff
@@ -11,6 +11,7 @@
     <NpmBuild Condition=" '$(Configuration)' == 'Debug' ">build-dev</NpmBuild>
     <NpmBuild Condition=" '$(NpmBuild)' == '' ">build</NpmBuild>
     <DevServerRunning>false</DevServerRunning>
+    <UserSecretsId>5B9ECED8-0E0C-443E-98B1-F1E508AB292B</UserSecretsId>
```

Comment:
If we instead put this in a `Directory.Build.props` file, then we don't have to have it duplicated in multiple projects, and in the `.Data.Test` source code.